### ISSUE TITLE
Clicking on emoji selector makes conversations disappear, with error in console

### ISSUE THREAD

**Issue by cronco** [sindresorhus/caprine#680]
Following stack trace comes up in the dev console on clicking:

```
ErrorUtils caught an error: "<![EX[["Tried to dispatch action %s while already dispatching %s from %s: %s"...". Subsequent errors won't be logged; see https://fburl.com/debugjs.
reportError @ BUl6rx8AmyS.js:55
k.$FBLogMessage1 @ BUl6rx8AmyS.js:96
k.mustfix @ BUl6rx8AmyS.js:96
j @ o2rxJlTy5Z7.js:74
a.dispatch @ o2rxJlTy5Z7.js:74
updateTrayData @ 3kohQXyp6C0.js:56
addMRUCategory @ 3kohQXyp6C0.js:61
getMRUCategory @ 3kohQXyp6C0.js:61
a.getState @ 3kohQXyp6C0.js:62
(anonymous) @ 3kohQXyp6C0.js:715
(anonymous) @ wktsn3-kjDj.js:61
i @ wktsn3-kjDj.js:61
(anonymous) @ wktsn3-kjDj.js:64
jf @ _X80TmlDpSY.js:13
kf @ _X80TmlDpSY.js:13
Cg @ _X80TmlDpSY.js:13
Hg @ _X80TmlDpSY.js:13
qh @ _X80TmlDpSY.js:13
rh @ _X80TmlDpSY.js:13
Zh @ _X80TmlDpSY.js:13
Xh @ _X80TmlDpSY.js:13
Sh @ _X80TmlDpSY.js:13
wh @ _X80TmlDpSY.js:13
enqueueSetState @ _X80TmlDpSY.js:13
a.setState @ n4rm-a-wPwZ.js:12
sendMethod @ wktsn3-kjDj.js:64
i @ wktsn3-kjDj.js:61
(anonymous) @ 3kohQXyp6C0.js:715
applyWithGuard @ BUl6rx8AmyS.js:55
a.__emitToSubscription @ BUl6rx8AmyS.js:61
a.emit @ BUl6rx8AmyS.js:61
a.__inform @ ri0lnrotWD5.js:97
a.__invokeOnDispatch @ ri0lnrotWD5.js:97
(anonymous) @ ri0lnrotWD5.js:97
a.__invokeCallback @ o2rxJlTy5Z7.js:74
a.$ExplicitRegistrationDispatcher4 @ o2rxJlTy5Z7.js:76
a.__invokeCallback @ o2rxJlTy5Z7.js:74
a.$7 @ o2rxJlTy5Z7.js:74
a.dispatch @ o2rxJlTy5Z7.js:74
loadedTrayData @ 3kohQXyp6C0.js:56
(anonymous) @ 3kohQXyp6C0.js:61
I.guard.propagationType @ BUl6rx8AmyS.js:103
n @ BUl6rx8AmyS.js:103
l @ BUl6rx8AmyS.js:181
(anonymous) @ BUl6rx8AmyS.js:181
applyWithGuard @ BUl6rx8AmyS.js:55
n @ BUl6rx8AmyS.js:103
g.(anonymous function) @ BUl6rx8AmyS.js:149
r @ BUl6rx8AmyS.js:149
a @ BUl6rx8AmyS.js:149
postMessage (async)
l @ BUl6rx8AmyS.js:149
c [from setImmediatePolyfill] @ BUl6rx8AmyS.js:150
a [from setImmediateAcrossTransitions] @ BUl6rx8AmyS.js:151
p @ BUl6rx8AmyS.js:181
s @ BUl6rx8AmyS.js:181
q @ BUl6rx8AmyS.js:181
(anonymous) @ BUl6rx8AmyS.js:181
(anonymous) @ 3kohQXyp6C0.js:58
AsyncRequest._dispatchResponse @ 9ZtjyVQyF32.js:23
applyWithGuard @ BUl6rx8AmyS.js:55
d @ BUl6rx8AmyS.js:55
(anonymous) @ 9ZtjyVQyF32.js:21
e @ BUl6rx8AmyS.js:185
applyWithGuard @ BUl6rx8AmyS.js:55
n @ BUl6rx8AmyS.js:103
setTimeout (async)
c [from setTimeoutAcrossTransitions] @ BUl6rx8AmyS.js:152
a [from setTimeout] @ BUl6rx8AmyS.js:185
(anonymous) @ 9ZtjyVQyF32.js:23
applyWithGuard @ BUl6rx8AmyS.js:55
a.registerCallback @ BUl6rx8AmyS.js:56
Y @ BUl6rx8AmyS.js:153
loadResources @ BUl6rx8AmyS.js:153
AsyncRequest.invokeResponseHandler @ 9ZtjyVQyF32.js:23
AsyncRequest._handleXHRResponse @ 9ZtjyVQyF32.js:23
(anonymous) @ 9ZtjyVQyF32.js:23
I.guard.propagationType @ BUl6rx8AmyS.js:103
applyWithGuard @ BUl6rx8AmyS.js:55
n @ BUl6rx8AmyS.js:103
f.last @ BUl6rx8AmyS.js:100
(anonymous) @ 9ZtjyVQyF32.js:23
XMLHttpRequest.send (async)
b @ XmEKLJtHNXT.js:10
AsyncRequest.send @ 9ZtjyVQyF32.js:23
(anonymous) @ 3kohQXyp6C0.js:58
m @ BUl6rx8AmyS.js:181
u @ BUl6rx8AmyS.js:181
n [from Promise] @ BUl6rx8AmyS.js:181
g @ 3kohQXyp6C0.js:58
fetchTrayData @ 3kohQXyp6C0.js:58
onTrayDataReady @ 3kohQXyp6C0.js:61
_loadTrayData @ AUZLsCaAtI7.js:5
_onShown @ AUZLsCaAtI7.js:5
componentDidMount @ AUZLsCaAtI7.js:5
(anonymous) @ n4rm-a-wPwZ.js:16
$h @ _X80TmlDpSY.js:13
Zh @ _X80TmlDpSY.js:13
Xh @ _X80TmlDpSY.js:13
Sh @ _X80TmlDpSY.js:13
wh @ _X80TmlDpSY.js:13
enqueueSetState @ _X80TmlDpSY.js:13
a.setState @ n4rm-a-wPwZ.js:12
(anonymous) @ vDYcZxofTwS.js:87
a @ BUl6rx8AmyS.js:119
f @ BUl6rx8AmyS.js:153
(anonymous) @ BUl6rx8AmyS.js:153
n @ BUl6rx8AmyS.js:103
A @ BUl6rx8AmyS.js:38
y @ BUl6rx8AmyS.js:38
H @ BUl6rx8AmyS.js:38
L @ BUl6rx8AmyS.js:38
a @ BUl6rx8AmyS.js:153
applyWithGuard @ BUl6rx8AmyS.js:55
a.$6 @ BUl6rx8AmyS.js:56
a.satisfyPersistentDependency @ BUl6rx8AmyS.js:56
(anonymous) @ BUl6rx8AmyS.js:153
done @ BUl6rx8AmyS.js:153
f @ BUl6rx8AmyS.js:153
(anonymous) @ BUl6rx8AmyS.js:153
applyWithGuard @ BUl6rx8AmyS.js:55
a.registerCallback @ BUl6rx8AmyS.js:56
Y @ BUl6rx8AmyS.js:153
loadResources @ BUl6rx8AmyS.js:153
(anonymous) @ BUl6rx8AmyS.js:153
n @ BUl6rx8AmyS.js:103
(anonymous) @ BUl6rx8AmyS.js:153
I.guard.propagationType @ BUl6rx8AmyS.js:103
applyWithGuard @ BUl6rx8AmyS.js:55
n @ BUl6rx8AmyS.js:103
h.onreadystatechange @ BUl6rx8AmyS.js:153
XMLHttpRequest.send (async)
b @ XmEKLJtHNXT.js:10
ia @ BUl6rx8AmyS.js:153
X @ BUl6rx8AmyS.js:153
(anonymous) @ BUl6rx8AmyS.js:153
Y @ BUl6rx8AmyS.js:153
loadModules @ BUl6rx8AmyS.js:153
a.loadAll @ o2rxJlTy5Z7.js:81
(anonymous) @ vDYcZxofTwS.js:88
a.UNSAFE_componentWillMount @ vDYcZxofTwS.js:87
gg @ _X80TmlDpSY.js:13
Cg @ _X80TmlDpSY.js:13
Hg @ _X80TmlDpSY.js:13
qh @ _X80TmlDpSY.js:13
rh @ _X80TmlDpSY.js:13
Zh @ _X80TmlDpSY.js:13
Xh @ _X80TmlDpSY.js:13
di @ _X80TmlDpSY.js:13
hd @ _X80TmlDpSY.js:13
n @ BUl6rx8AmyS.js:103
applyWithGuard @ BUl6rx8AmyS.js:55
fire @ BUl6rx8AmyS.js:262
(anonymous) @ BUl6rx8AmyS.js:262
(anonymous) @ BUl6rx8AmyS.js:238
I.guard.propagationType @ BUl6rx8AmyS.js:103
n @ BUl6rx8AmyS.js:103
(anonymous) @ BUl6rx8AmyS.js:238
applyWithGuard @ BUl6rx8AmyS.js:55
n @ BUl6rx8AmyS.js:103
(anonymous) @ BUl6rx8AmyS.js:102
```

**Comment by dusansimic** [comment]
This seems to be a problem on messenger.com since it doesn't work in the browser either.

**Comment by cronco** [comment]
@dusansimic it works for me

**Comment by dusansimic** [comment]
Seems like it's fixed now.
We can close this.

**Comment by mrwnmonm** [comment]
I still have the issue

**Comment by CvX** [comment]
Fixed in #941
